### ISSUE TITLE
Offline mode — start without server connection

### ISSUE THREAD

**Issue by Twilightssuperb** [altairwei/WizNotePlus#233]
In readme I saw plans for fully offline portable client. Portable version is already available.

Just ask — how long should I wait for offline mode?

**Comment by altairwei** [comment]
Offline mode is planned for v2.15, but it's not likely to be implemented for at least six months, as I'm very busy at the moment.

**Comment by Twilightssuperb** [comment]
What if to just implement a simple solution — option to choose «No server connection, use local data storage» at first launch?

**Comment by altairwei** [comment]
> What if to just implement a simple solution — option to choose «No server connection, use local data storage» at first launch?

It seems easy to implement, but how to authenticate user accounts? By the way, there are many features won't work in this simple solution, including full text search, document history and so on.

**Comment by altairwei** [comment]
> at first launch?

And what you mean "at first launch"? new account creation?

**Comment by Twilightssuperb** [comment]
> new account creation?

Yep. Maybe, local (offline) account without authentication etc. is not a real offline mode, but it's a kind of compromise — local account instead of (future) offline mode.

Sorry, I don't know technical details for how offline mode would work. I just suppose easier way...

**Comment by altairwei** [comment]
It will be very easy to implement an offline mode (a simple solution) for an existing account. Hoever, creating an account without network will be much more tricky. I plan to investigate the possibility of implementing this simple mode in v2.12
